src/Connection.php

```diff
@@ -287,13 +289,18 @@ public function publish($subject, $payload)
      *
      * @param string   $subject  Message topic.
      * @param \Closure $callback Closure to be executed as callback.
+     * @param string   $queue    Queue name for subscription.
      *
      * @return string
      */
-    public function subscribe($subject, \Closure $callback)
+    public function subscribe($subject, \Closure $callback, $queue = null)
```

Comment:
Maybe implementation a new method queueSubscribe can help reading code, and allow the have a better method signature:

`public function queueSubscribe($subject, $queue, \Closure $callback)`